docs/installation.md

```diff
@@ -97,6 +97,20 @@ docker run -p 9090:9090 my-prometheus
 A more advanced option is to render the configuration dynamically on start
 with some tooling or even have a daemon update it periodically.
 
+### Overriding command line parameters
+
+The Docker image is started with a number of default command line parameters:
```

Comment:
Thanks for this.
To prevent any divergence if the Dockerfile changes in the future I wouldn't explicitly put the args here, I'd just hint that the Dockerfile corresponding to the release/version should be checked.